File: 4st/Hani/programmer/로그인 성공?.java
class Solution {
    public String solution(String[] id_pw, String[][] db) {
        String answer = "";
        for (int i = 0; i < db.length; i++) {
            if (id_pw[0].equals(db[i][0])) {
                if (id_pw[1].equals(db[i][1])) {
                    answer = "login";
                }
                else {
                    answer = "wrong pw";
                }
            }
        }
        if (answer == "") {
            answer = "fail";
        }
        return answer;
    }
}

File: 4st/Hani/programmer/로그인 성공?.js
function solution(id_pw, db) {
  for (const [id, pw] of db) {
    if (id === id_pw[0]) {
      if (pw === id_pw[1]) {
        return "login";
      } else {
        return "wrong pw";
      }
    }
  }
  return "fail";
}
